testsuite/integration/smoke/src/test/java/org/jboss/as/test/smoke/jms/JMSBridgeTest.java

```diff
@@ -125,6 +136,8 @@ public void sendAndReceiveMessage() throws Exception {
             assertTrue(receivedMessage instanceof TextMessage);
             assertEquals(text, ((TextMessage) receivedMessage).getText());
             assertNotNull("did not get header set by the JMS bridge", receivedMessage.getStringProperty(ActiveMQJMSConstants.AMQ_MESSAGING_BRIDGE_MESSAGE_ID_LIST));
+             assertEquals("Message count bridge metric is not correct", 5000L, readMetric("message-count"));
```

Comment:
@ehsavoie Why is this 5000L and not 1L?

Reply:
Because a NPE was thrown when getting the value which is 1L. Test fixed